templates/overrides/page/page.html.twig

```diff
@@ -115,8 +115,7 @@
       {% endif %}
     </header>
   {% endblock %}
-  <main aria-label="{{ "Main content"|t }}">
-    <a id="main-content" tabindex="-1"></a>
+  <main aria-label="{{ "Main content"|t }}" id="main-content" tabindex="-1">
```

Comment:
One observation I make:
When the `<main>` element becomes active/focused, it gets `:focus-visible` class and gets an outline.
That outline looks a bit ugly (browser native styling).
Maybe it's ok for now, given that we have no other design specs.
So I'd say let's keep it, but we might get some interesting feedback.

Reply:
The issue described by @donquixote is a direct result of how tabindex="-1" affects the `<main>` element focus. Normally, a <main> element is not focusable. When a user clicks "Skip to main content" link, the browser scrolls to the element but does not transfer focus to it.

Adding `tabindex="-1"` makes the `<main>` element focusable so when the skip link is clicked, the browser moves the keyboard focus from the link to the `<main>` element, using the :focus-visible state, which is why the blue outline appears.